polkadot-protocol/protocol-components/.pages

```diff
@@ -0,0 +1,7 @@
+title: Protocol Components
```

Comment:
There isn't a protocol components section in the new IA, so I think this can be deleted